package.json

```diff
@@ -64,6 +64,7 @@
     "express": "^4.17.1",
     "glob": "^7.1.7",
     "google-auth-library": "^8.5.2",
+    "googleapis": "^110.0.0",
```

Comment:
Please pin newly introduced dependencies (remove the `^`) to not rely on package-lock.json shenanigans.